src/main/java/UITest/InvalidCredentials.java

```diff
@@ -0,0 +1,26 @@
+package UITest;
+
+public enum InvalidCredentials {
+    WRONG_PASSWORD("admin", "wrongPassword"),
+    WRONG_USERNAME("wrongUser", "admin"),
+    BOTH_WRONG("ababahalamaga", "wrongPassword"),
+    EMPTY_PASSWORD("admin", " "),
+    BOTH_EMPTY(" ", " "),
+    EMPTY_USERNAME(" ", "admin"),;
+
+    private final String username;
+    private final String password;
+
+    InvalidCredentials(String username, String password) {
```

Comment:
you can replace constructor and getters with @AllArgsConstructor and @Getter annotations from Lombok